fdbclient/VersionVector.h

```diff
@@ -144,7 +146,59 @@ struct VersionVector {
 
 	template <class Ar>
 	void serialize(Ar& ar) {
-		serializer(ar, versions, maxVersion);
+		// Encoded version of the "version vector":
+		//
+		// Encode localities by using RLE: Capture localities and their run lengths.
+		// Note that the order the localities appear in this map matches the order that they appear
+		// in the version vector.
+		std::map<int8_t, int8_t> tagLocalities;
```

Comment:
`int8_t` for the count seems small. I think we need at least `int16_t`, the same type as tag ID.

Reply:
Ok, done.